utilities/ubi8-google-api-python-client/scripts/upload-file-to-google-drive.py

```diff
@@ -0,0 +1,263 @@
+from __future__ import print_function
```

Comment:
Since this is meant to be run with python3, it should start with:
```
#!/usr/bin/env python3
```

Reply:
@jkupferer i didn't put that in since some systems don't have `python3` they just have `python` which happens to be `python3` I didn't want to hard code in any script paths and thought it better to just have people run it using whichever python library they had wherever they had it. not to mention the complications when using virtual environments.

Reply:
updated to to do this in latest push.